aerogear/mobile-security-service
# feat(commit-message-check): adding bash script for checking commit messages to circle ci (AEROGEAR-8465)

## Motivation
adding bash script for checking commit messages to circle ci 
https://issues.jboss.org/browse/AEROGEAR-8465

## What
adding bash script for checking commit messages to circle ci 

## Why
to meet with the guide lines outline here
https://github.com/fheng/help/blob/master/development_process.md#commit-messages

## How
 adding bash script for checking commit messages to circle ci 
## Verification Steps
Add the steps required to check this change. Following an example.
If this PR passes the circle ci build the you can consider it working


## Checklist:


## Progress
